generators/src/new-test-model/new-test-model.generator.ts

```diff
@@ -1,6 +1,8 @@
 import { execSync } from 'node:child_process';
-import { existsSync, mkdirSync, writeFileSync } from 'node:fs';
+import { constants } from 'node:fs';
+import { access, mkdir, writeFile, readFile, cp } from 'node:fs/promises';
```

Comment:
Since we are adjusting this file, we might as well improve a bunch of other things (e.g. using async functions).